GitUI/UserControls/FileStatusList.cs

```diff
@@ -91,6 +99,15 @@ public FileStatusList()
 
             _fullPathResolver = new FullPathResolver(() => Module.WorkingDir);
             _revisionTester = new GitRevisionTester(_fullPathResolver);
+            _noItemStatuses = new[]
+            {
+                new GitItemStatus
+                {
+                    Name = $"     - {NoFiles.Text} -",
+                    IsStatusOnly = true,
+                    ErrorMessage = ""
```

Comment:
`ErrorMessage` should be assigned either `null`, i.e. not assigned, or `string.Empty`, shouldn't it?

Reply:
Empty string has a special meaning: No icon in the status list.